### ISSUE TITLE
Error while using with create-react-app

### ISSUE THREAD

**Issue by Rhymond** [reactstrap/reactstrap#505]
### Issue description

Error:
```
./~/bootstrap/dist/css/bootstrap.css
Module build failed: TypeError: Path must be a string. Received undefined
```
I installed `reactstrap`  using these commands:
```
npm install bootstrap@4.0.0-alpha.6 --save
npm install --save reactstrap react-addons-transition-group react-addons-css-transition-group react react-dom
```
And imported boostrap
```
import 'bootstrap/dist/css/bootstrap.css';
```

- version `^4.8.0`

**Comment by Rhymond** [comment]
Found that `create-react-app` can't generate CSS files with `@supports` tag

**Comment by pongsakornsemsuwan** [comment]
come here for the same issue

**Comment by TheSharpieOne** [comment]
facebookincubator/create-react-app#2836

> Looks like this was fixed a few days ago: postcss/autoprefixer#865; ai/browserslist#158; ai/browserslist#161. Please reinstall your node_modules/ and let us know if this is still happening.

Closing since this is not directly related to reactstrap
